frontend/app/components/vertical-collection/component.js

```diff
@@ -1,11 +1,9 @@
 import { isTesting, macroCondition } from "@embroider/macros";
 import VerticalCollectionComponent from "@html-next/vertical-collection/components/vertical-collection/component";
-import classic from "ember-classic-decorator";
 
-@classic
 export default class VerticalCollection extends VerticalCollectionComponent {
-  init(...args) {
-    super.init(...args);
```

Comment:
This still extends a [classic component](https://github.com/html-next/vertical-collection/blob/a52ab83185729f6f7cb7024ed5b904c977ea4bd7/addon/components/vertical-collection/component.js#L15)!?